test/test_processing_utils.py

```diff
@@ -361,13 +361,13 @@ def test_add_root_url():
     new_expected = {
```

Comment:
This test was wrong which is why we didn't notice this bug sooner (this is what I get for not paying close enough attention to GH Copilot's output)